templates/template-dominator/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+#![allow(dead_code)]
+#![allow(warnings)]
+
+pub mod app;
+use wasm_bindgen::JsValue;
+use std::sync::Arc;
+use wasm_bindgen::prelude::*;
+
+use crate::{app::App};
+
+#[wasm_bindgen(start)]
+pub async fn run() -> Result<(), JsValue> {
+    init_logger();
+    std::panic::set_hook(Box::new(on_panic));
```

Comment:
let's remove all things related to loggings and keep the template as simple as possible